src/js/rx/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "Prescription Refill",
+  "entryFile": "./rx-entry.jsx",
+  "entryName": "rx",
+  "rootUrl": "/health-care/prescriptions",
+  "production": true
+}
```

Comment:
So when one of these files is edited/added, Jenkins will process it and automatically submit a PR to the rev proxy? Lol